### PR TITLE
Add drupal-nginx-fpm:0.46

### PR DESCRIPTION
**Pull request by LeonZhang77**

### CODE REVIEW

File: drupal-nginx-fpm/0.46/php.ini [hunk 1]
```diff
@@ -0,0 +1,25 @@
+; http://php.net/manual/en/errorfunc.configuration.php
+log_errors=On
+error_log=/var/log/httpd/php-error.log
```
Comment (EricSten-MSFT) [inline drupal-nginx-fpm/0.46/php.ini]: Should this be in one of the folders which survives past the container lifetime?  Maybe /home/LogFiles/php-error.log ?

**Comment by EricSten-MSFT** [comment]
One general comment for the code review: We really need to have logrotate handling any of the error logs.  Sometimes these images stay up for weeks, and the log files get really huge.  It would be great if we could add some logrotate settings in the entrypoint.sh.  This also might require us to add ```apt-get install -q -y logrotate```, and add some logrotate config files that should get copied to ```/etc/logrotate.d``` directory.

I'm thinking of the nginx, php, sql and drupal logs.

**Comment by EricSten-MSFT** [comment]
> I will try to add logrotate into next version. is it OK?

Totally fine.  Please open an Issue to track this feature work.  Thx!

File: drupal-nginx-fpm/0.46/php.ini (hunk 1 above)
Comment (LeonZhang77) [reply drupal-nginx-fpm/0.46/php.ini]: PR: https://github.com/Azure/app-service-quickstart-docker-images/pull/192.

**Pull request merged**